endpoints/address/tests/integration_test.js

```diff
@@ -9,3 +10,27 @@ describe('zip', () => {
     request.get(params, resultHandler)
   })
 })
+
+describe('address graphql', () => {
+  it('should return only the field that were asked for', (done) => {
+    const expected = {
+      data: {
+        address: [
+          { street: 'laugavegur', house: '1' },
+        ],
+      },
+    }
+    const params = helpers.testRequestParams('/graphql', { query: `
+      query {
+        address(address: "laugavegur 1") {
+          street
+          house
+        }
+      }
+    ` })
+    request.get(params, (err, res, body) => {
+      assert(body === JSON.stringify(expected), 'result will match expected')
```

Comment:
This assertion is not ideal. If the string don't match the assertion will fail with 
```
      Uncaught AssertionError: result will match expected
      + expected - actual

      -false
      +true
```
which is not helpful. We should use more sophisticated assertion. Object based assertion would probably be best. It's more robust agains party-crashers like mismatching white-spaces. Something like:

```js
areDeepEquals(JSON.parse(body), expected)
```

We should also consider creating an abstraction for GraphQL testing. It should be straight forward since GraphQL tests mostly just compare JS objects.